app/uk/gov/hmrc/mtdsatestsupportapi/connectors/DeleteVendorStateConnector.scala

```diff
@@ -16,31 +16,35 @@
 
 package uk.gov.hmrc.mtdsatestsupportapi.connectors
 
+import api.connectors.DownstreamUri
+import api.connectors.httpparsers.StandardDownstreamHttpParser
 import api.connectors.{BaseDownstreamConnector, DownstreamOutcome}
 import config.AppConfig
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
-import api.connectors.httpparsers.StandardDownstreamHttpParser.readsEmpty
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, StringContextOps}
+import uk.gov.hmrc.mtdsatestsupportapi.models.request.deleteStatefulTestData.DeleteStatefulTestDataRequest
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
-import api.connectors.DownstreamUri._
-import uk.gov.hmrc.mtdsatestsupportapi.models.request.deleteStatefulTestData.DeleteStatefulTestDataRequest
 
-class DeleteVendorStateConnector @Inject() (val http: HttpClient, val appConfig: AppConfig) extends BaseDownstreamConnector {
+class DeleteVendorStateConnector @Inject() (val http: HttpClientV2, val appConfig: AppConfig)
+    extends BaseDownstreamConnector
+    with StandardDownstreamHttpParser {
 
   def deleteVendorState(request: DeleteStatefulTestDataRequest)(implicit
       hc: HeaderCarrier,
       ec: ExecutionContext,
       correlationId: String): Future[DownstreamOutcome[Unit]] = {
     import request._
 
-    val queryParams: Seq[(String, String)] = nino match {
-      case Some(n) => Seq(("taxableEntityId", n.toString))
-      case None    => Seq()
+    implicit val context: ConnectorContext = ConnectorContext(appConfig.stubDownstreamConfig)
+
+    val url = nino match {
+      case Some(n) => url"${context.baseUrl}/test-support/vendor-state/$vendorClientId?taxableEntityId=${n.value}"
```

Comment:
You might be able to use the constructor: 
`
new URL(new URL("SOME_BASE_URL"), "/PATH1/PATH2") 
`
in the BaseDownstreamConnector. 

This will let you separate out the base url in the connectors and just have the path URI here instead